website/tutorial/basic/08-authentication.mdx

```diff
@@ -0,0 +1,597 @@
+---
```

Comment:
General comment on this page:

I like that we provide Authentication 101 but I don't feel like it's related to GraphQL and make the tutorial much longer and especially requires more attention to follow through.
There are plenty of libraries that handle the JWT flow, especially the Envelop's https://www.envelop.dev/plugins/use-generic-auth

Why not showcase Envelop and how to use https://www.envelop.dev/plugins/use-generic-auth?